mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

```diff
@@ -164,23 +165,40 @@ def select(self, query: ast.Select) -> pd.DataFrame:
 
     @staticmethod
     def clean_selected_columns(selected_cols) -> None:
```

Comment:
This function and the call to it above is no longer needed, because this is removing the id column and then appending it again to the list.